### PR TITLE
Release v2.3.1

### PR DESCRIPTION
**Pull request by pbobov**

### CODE REVIEW

**Comment by pbobov** [comment]
@JoAlvarez, @jonathanan Please review the changes.

**Comment by jonathanan** [comment]
@pbobov Changes look good.

File: cookbooks/arcgis-geoevent/CHANGELOG.md [hunk 1]
```diff
@@ -3,7 +3,12 @@ arcgis-geoevent cookbook CHANGELOG
...
+2.3.1
+-----
+- Configure 'arcgisgeoevent' system.d service.
+- Added support for ArcGIS 10.5.
+
 2.3.0
 -----
 - Split from 'arcgis' 2.2.1 cookbook.
-- Added support for ArcGIS 10.5.
+- Added support for ArcGIS 10.5 Beta.
```
Comment (JoAlvarez) [inline cookbooks/arcgis-geoevent/CHANGELOG.md]: Should this have Beta listed still?

File: cookbooks/arcgis-server/recipes/sql_alias.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
+#
+# Cookbook Name:: arcgis-server
+# Recipe:: sql_alias
+#
+# Copyright 2015 Esri
```
Comment (JoAlvarez) [inline cookbooks/arcgis-server/recipes/sql_alias.rb]: File created and published this year, should have this year listed for copyright

File: cookbooks/arcgis-server/recipes/server_node.rb [hunk 3]
```diff
@@ -57,8 +58,8 @@
 arcgis_server_server 'Authorize ArcGIS Server' do
   authorization_file node['arcgis']['server']['authorization_file']
   authorization_file_version node['arcgis']['server']['authorization_file_version']
-  retries 5
-  retry_delay 60
+#  retries 5
```
Comment (JoAlvarez) [inline cookbooks/arcgis-server/recipes/server_node.rb]: Can lines 61 and 62 be removed here?

File: cookbooks/arcgis-server/recipes/portal.rb [hunk 4]
```diff
@@ -103,7 +103,11 @@
   security_question node['arcgis']['portal']['security_question']
   security_question_answer node['arcgis']['portal']['security_question_answer']
   install_dir node['arcgis']['portal']['install_dir']
-  content_dir node['arcgis']['portal']['content_dir']
+  #content_dir node['arcgis']['portal']['content_dir']
```
Comment (JoAlvarez) [inline cookbooks/arcgis-server/recipes/portal.rb]: Can this line be removed?

File: cookbooks/arcgis-pro/CHANGELOG.md [hunk 5]
```diff
@@ -3,7 +3,11 @@ arcgis-pro cookbook CHANGELOG
...
 This file is used to list changes made in each version of the arcgis-pro cookbook.
 
+2.3.1
+-----
+- Added support for ArcGIS Pro 1.3.
+
 2.3.0
 -----
-- This version supports ArcGIS Pro 11, 1.2, and 1.3.
+- This version supports ArcGIS Pro 1.1, 1.2, and 1.3 Beta.
```
Comment (pbobov) [inline cookbooks/arcgis-pro/CHANGELOG.md]: That is the changelog of 2.3.0 release.

File: cookbooks/arcgis-server/attributes/geoanalytics.rb [hunk 6]
```diff
@@ -0,0 +1,25 @@
+#
+# Cookbook Name:: arcgis-server
+# Attributes:: geoanalytics
+#
+# Copyright 2015 Esri
```
Comment (JoAlvarez) [inline cookbooks/arcgis-server/attributes/geoanalytics.rb]: File created and published should list year they were created for copyright

**Pull request merged**